gemini/use-cases/applying-llms-to-data/analyze-poster-images-in-bigquery/poster_image_analysis.ipynb

```diff
@@ -0,0 +1,708 @@
+{
```

Comment:
<p>Can you add the buttons used in other notebooks for this repo?</p><p> </p><p>Similar to <a href="https://github.com/GoogleCloudPlatform/generative-ai/blob/main/gemini/getting-started/intro_gemini_python.ipynb" target="_blank">https://github.com/GoogleCloudPlatform/generative-ai/blob/main/gemini/getting-started/intro_gemini_python.ipynb</a> ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/511/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>There's isn't yet a "Run in BigQuery Studio" button though it is in the works. I really only want people to open in BigQuery Studio, so ideally that would be the only button. I can add an open in Colab Enterprise button if you want?</p>

Reply:
<p>I just added a "Run in Colab Enterprise" button, but plan to keep this paragraph. That work?</p>

Reply:
Thats good